Documentation/doc/resources/1.9.6/header.html

```diff
@@ -1,6 +1,6 @@
 <!-- HTML header for doxygen 1.9.6-->
 <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "https://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html xmlns="http://www.w3.org/1999/xhtml" lang="$langISO">
+<html xmlns="http://www.w3.org/1999/xhtml" lang="$langISO" class="light-mode">
```

Comment:
I did not check but since we have custom headers this option will not be taken into account.

Reply:
The fact that CGAL uses custom headers is complicating things a bit, I still think that it will be a good idea to set:
```
HTML_COLORSTYLE        = LIGHT
```
as in that case some default files will be adjusted as well and might prevent some problems in the future.

I did a quick test with the Manual directory and e.g. in the file `dynsections.js` we see the differences:
```
127,128c127,128
<   plusImg:  [ "var(--fold-plus-image)",  "var(--fold-plus-image-relpath)" ],
<   minusImg: [ "var(--fold-minus-image)", "var(--fold-minus-image-relpath)" ],
---
>   plusImg:  [ "url('plus.svg')",  "url('../../plus.svg')" ],
>   minusImg: [ "url('minus.svg')", "url('../../minus.svg')" ],
156c156
<       'background':'linear-gradient(var(--fold-line-color),var(--fold-line-color)) no-repeat 46px/2px 100%'
---
>       'background':'linear-gradient(#808080,#808080) no-repeat 46px/2px 100%'
```
(lines `<` are without the setting `HTML_COLORSTYLE        = LIGHT`).

Other files with differences are in `.css` file:
- `doxygen.css`
- `navtree.css`
- `search/search.css`
- `tabs.css`

Not performed changes here might lead to unexpected results in the layout / coloring when `var(...)` are not replaced.

Reply:
Besides the mentioned files I didn't find evidence that the light switch was used elsewhere.
I also searched for the word light and found no evidence that it would lead to problems.
So I thin the setting `HTML_COLORSTYLE        = LIGHT` can be used, unless you can point to a place where the setting leads to problems and the `class="light-mode"` doesn't.